bridge/cypress/support/pageobjects/UniformPage.ts

```diff
@@ -353,6 +353,33 @@ class UniformPage {
     return this;
   }
 
+  public assertSecretCreationLink(): this {
+    cy.byTestId('ktb-webhook-secret-selector').first().click();
+    cy.byTestId('ktb-webhook-secret-creation-link').should('exist');
+    return this;
+  }
+
+  public createSecret(): this {
```

Comment:
`createSecret` is not correctly named here. I would assume that a secret is created, but instead it redirects to the add-secret page through the link in the value selector